src/mlpack/tests/range_search_test.cpp

```diff
@@ -487,13 +486,13 @@ BOOST_AUTO_TEST_CASE(DualTreeVsNaive1)
 
   for (size_t i = 0; i < sortedTree.size(); ++i)
   {
-    BOOST_REQUIRE(sortedTree[i].size() == sortedNaive[i].size());
+    REQUIRE(sortedTree[i].size() == sortedNaive[i].size());
 
     for (size_t j = 0; j < sortedTree[i].size(); ++j)
     {
-      BOOST_REQUIRE(sortedTree[i][j].second == sortedNaive[i][j].second);
-      BOOST_REQUIRE_CLOSE(sortedTree[i][j].first, sortedNaive[i][j].first,
-          1e-5);
+      REQUIRE(sortedTree[i][j].second == sortedNaive[i][j].second);
+      REQUIRE(sortedTree[i][j].first ==
+              Approx(sortedNaive[i][j].first).epsilon(1e-7));
```

Comment:
Let's be consistent with the style guide here, tab twice (4 spaces), this applies to a couple other lines down below as well.

Reply:
```c++
REQUIRE(sortedTree[i][j].first ==
    Approx(sortedNaive[i][j].first).epsilon(1e-7));
```
Hey @zoq , Just to be sure you mean something like this right ?

Reply:
Also i am a bit confused about when can we use parent parenthesis indentation and when we should follow the 2 tabs guide, like i guess the tab 2 guide is more prominent, so what are some cases for the first case ?

Reply:
For functions it makes sense to align the parameter, but for everything else I would stick to the later, I'm sure there are places where the first is used, and that's okay.

Reply:
Got you, thanks.